src/blackboxes.cc

```diff
@@ -224,6 +224,7 @@ void export_blackbox_to_rtlil(ast::Compilation &comp, const ast::InstanceSymbol
 	}
 
 	RTLIL::Module *mod = target->addModule(name);
+	mod->set_bool_attribute(ID(blackbox), true);
```

Comment:
This makes me think `--empty-blackboxes` did not fully work until now

Reply:
It's supposed to enable automatically considering empty modules for blackboxes, same as Verific or Yosys read_verilog behavior.

Reply:
I think Yosys has that behavior built-in, but it might be in the Verilog frontend rather than in the RTLIL itself